pkg/operator/ceph/cr_manager.go

```diff
@@ -34,13 +36,13 @@ func (o *Operator) startManager(namespaceToWatch string, stopCh <-chan struct{})
 	logger.Info("setting up the controller-runtime manager")
 	kubeConfig, err := config.GetConfig()
 	if err != nil {
-		logger.Errorf("failed to get client config for controller-runtime manager. %v", err)
+		mgrErrorCh <- errors.Wrap(err, "failed to get client config for controller-runtime manager.")
```

Comment:
```suggestion
		mgrErrorCh <- errors.Wrap(err, "failed to get client config for controller-runtime manager")
```

Reply:
Reflected all suggestions. My patch got far better than the initial version, thanks.